pkg/kubelet/kubelet.go

```diff
@@ -1516,6 +1548,29 @@ func (kl *Kubelet) canAdmitPod(pods []*api.Pod, pod *api.Pod) (bool, string, str
 	return true, "", ""
 }
 
+func (kl *Kubelet) canRunPod(pod *api.Pod) lifecycle.PodAdmitResult {
```

Comment:
Ack. I left a TODO to get rid of that other method. Do you have a suggestion for a better name?